service/service.hpp

```diff
@@ -164,15 +164,19 @@ namespace raven
     {
         auto cfg = fill_request<config_create>(json_data);
         auto config_create_db_result = db_.config_create(json_data);
-        if (config_create_db_result.config_id.value() != static_cast<int>(request_state::db_error)) {
-            config_clients_registry_.at(sock.fileno()) += config_create_db_result.config_id;
+        if (!config_create_db_result.state_error.has_value()) {
+            using namespace std::string_literals;
+            auto unique_id = std::hash<std::string>()(cfg.config_name + "_"s + std::to_string(sock.fileno()) + "_"s +
+                std::to_string(config_create_db_result.config_id.value()));
+            config_clients_registry_.at(sock.fileno()) += std::make_pair(config_id_st{unique_id},
```

Comment:
Why do you do the association in create_config ? Shouldn't this happen in load_config ?

Reply:
No, we need to associate it, internally, config_create return the db id, this is perfect to make the association with the unique id.

Reply:
this isn't completly related to this matter, but I wonder if create config should also load the config once it's created ?

Reply:
Well, this doesn't work if the config is loaded when it wasn't created in this run of the service.
@Riz-cros Create_config should not load the config, it does not return the config_id.

Reply:
So the unique id is the key in that case ?

Reply:
So the unitque id is the key in that case ?